src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,37 @@
+.contact {
+
+  &__button {
+    margin-bottom: 48px;
+  }
+
```

Comment:
buttons should have different styles at default, hover, active
![btn](https://user-images.githubusercontent.com/89162624/209548446-5f60df54-ba82-44fa-98dd-7e116eea5de6.png)

Reply:
they have different style